Directory.Build.props

```diff
@@ -7,6 +7,7 @@
     <Product>Microsoft .NET</Product>
     <Copyright>$(CopyrightNetFoundation)</Copyright>
     <PackageLicenseExpression>MIT</PackageLicenseExpression>
+    <PackageProjectUrl>https://github.com/dotnet/templating</PackageProjectUrl>
```

Comment:
@marcpopMSFT What do you think here? Should the package project url point to dotnet/dotnet or dotnet/templating?

Reply:
The alternative is to point to https://dot.net which runtime sets.

Reply:
I don't think it matters a lot, but I agree it should either point to dotnet/templating or use the generic dot.net.

Reply:
I personally think that dot.net is probably right, but if we were going to shift this, we should do it all at once.